clang/docs/SYCLFERFCLoweringKernel.md

```diff
@@ -0,0 +1,84 @@
+# SYCL Kernel
+
+A SYCL construct such as `parallel_for` or 'single_task' takes a named function object or a lambda as one of its arguments.   The contents of this function object are executed on the device.   However, as the SYCL runtime can rely on other offload APIs like OpenCL or CUDA to execute the function object, it needs to respect the calling convention of these API.  To enable this, the function object is converted into the format of an OpenCL kernel.
```

Comment:
```suggestion
A SYCL construct such as `parallel_for` or `single_task` takes a named function object or a lambda as one of its arguments.   The contents of this function object are executed on the device.   However, as the SYCL runtime can rely on other offload APIs like OpenCL or CUDA to execute the function object, it needs to respect the calling convention of these API.  To enable this, the function object is converted into the format of an OpenCL kernel.
```

There are other cases where you have forward quotes rather than backwards ones (in this RFC and some of the others).  Read through the HTML render, and they are easier to spot.